web/backend-fastapi/app/models/trulens.py

```diff
@@ -81,66 +79,40 @@ def get_feedback_value(feedback):
         return 0
 
 
-def process_feedback(index, feedback, record_id=None, bulk=False):
```

Comment:
Why are we deleting this? This may break embedding adapters. Double check

Reply:
As far as I could see, it's not being used anywhere. There is a function with the same name in the feedback model which is used.